### PR TITLE
update changelog for 2.0

### PR DESCRIPTION
**Pull request by larskuhtz**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,46 @@
...
+This version replaces all previous versions. Any prior version will stop working
+on **2020-07-16T00:00:00Z**. Node administrators must upgrade to this version
+before that date.
+
+This version will stop working on **2020-08-13T00:00:00Z**.
+
+This version includes the fork that adds 10 additional chains to Chainweb
+resulting in 20 chains in total. The fork will occur on
+
+*   Testnet at block height 332,604, which is expect to happen around
```
Comment (mightybyte) [inline CHANGELOG.md:14]: What timezone are these?

Comment (larskuhtz) [reply CHANGELOG.md:14]: I think, it's UTC. We should add that.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,46 @@
...
+    2020-07-28 16:00:00, and
```
Comment (larskuhtz) [inline CHANGELOG.md:15]:
```suggestion
    2020-07-28 16:00:00 UTC, and
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,46 @@
...
+*   Mainnet at block height 852,054, which is expected to happen around
+    2020-08-20 16:00:00.
```
Comment (larskuhtz) [inline CHANGELOG.md:17]:
```suggestion
    2020-08-20 16:00:00 UTC.
```

File: CHANGELOG.md (hunk 1 above)
Comment (larskuhtz) [reply CHANGELOG.md:14]: done

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,46 @@
...
+
+Other changes:
+
+*   Full support for [Rosetta](https://www.rosetta-api.org).
+    (#1050, #1077, #1079, #1093)
+*   Support for Pact continuations in `local` queries. (#1080)
+*   Make Pact service more stable and fix a memory leak. (#1104, #1100)
+*   Fix a Pact bug related to module namespaces. (#1107)
+*   Make `info` endpoint show currently in use chains. (#1099)
+*   An improved difficult adjustment algorithm to support the graph transition
```
Comment (slpopejoy) [inline CHANGELOG.md:35]:
```suggestion
*   An improved difficulty adjustment algorithm to support the graph transition
```

**Pull request merged**